google/ads/google_ads/client.py

```diff
@@ -95,6 +95,12 @@ def load_from_string(cls, yaml_str):
         kwargs = cls._get_client_kwargs(config_data)
         return cls(**kwargs)
 
+    @classmethod  
+    def load_from_dict(cls,config_dict):
+	config_data=config.load_from_dict(config_dict)
```

Comment:
Also when I run the tests I get an error from this line about inconsistent use of tabs and spaces. Make sure to run the tests as you make updates to check for errors like this.

Reply:
I have made all the changes. I am facing ModuleNotFoundError: No module named 'google.protobuf' . Did they make changes to the google-ads package?

Used: pip install google-ads, pip install google, pip install protobuf